### PR TITLE
Add `device list --uuid <uuid>` support

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

**Comment by doanac** [comment]
@mike-sul based on:

https://git.foundries.io/development/cloudplatforms/infrastructure/ota-lite/merge_requests/66

File: client/foundries.go [hunk 1]
```diff
@@ -276,11 +278,17 @@ func (a *Api) setReqHeaders(req *http.Request, jsonContent bool) {
...
+		logrus.Debugf("Setting extra HTTP header %s=%s", k, v)
+		req.Header.Set(k, v)
+	}
+
 	if len(a.config.ClientCredentials.AccessToken) > 0 {
 		logrus.Debug("Using oauth token for http request")
 		tok := base64.StdEncoding.EncodeToString([]byte(a.config.ClientCredentials.AccessToken))
 		req.Header.Set("Authorization", "Bearer "+tok)
 	}
+	// req.Header.Set("OTA-LITE-SVC", "andy")
```
Comment (mike-sul) [inline client/foundries.go:291]: looks like a leftover?

File: client/foundries.go [hunk 2]
```diff
@@ -133,8 +133,9 @@ type FactoryUser struct {
 }
 
 type TargetStatus struct {
-	Devices int `json:"devices"`
-	Version int `json:"version"`
+	Devices      int `json:"devices"`
+	Version      int `json:"version"`
+	Reinstalling int `json:"(re-)installing"`
```
Comment (mike-sul) [inline client/foundries.go:138]: I guess it's a bit out of the 'uuid' topic, but I think it's OK :).

File: subcommands/status/cmd.go [hunk 3]
```diff
@@ -61,10 +61,10 @@ func showStatus(cmd *cobra.Command, args []string) {
 		}
 		fmt.Println("\n## Tag:", tag.Name)
 		// Tabby doesn't indent (or at least easily) so:
-		fmt.Println("\tTARGET  DEVICES  DETAILS")
-		fmt.Println("\t------  -------  -------")
+		fmt.Println("\tTARGET  DEVICES  (RE-)INSTALLING DETAILS")
+		fmt.Println("\t------  -------  --------------- -------")
 		for _, tgt := range tag.Targets {
-			fmt.Printf("\t%-6d  %-7d  `fioctl targets show %d`\n", tgt.Version, tgt.Devices, tgt.Version)
+			fmt.Printf("\t%-6d  %-7d  %-15d `fioctl targets show %d`\n", tgt.Version, tgt.Devices, tgt.Reinstalling, tgt.Version)
```
Comment (mike-sul) [inline subcommands/status/cmd.go:67]: Does `Reinstalling` print true or false or a list of apps are being reinstalled?

**Review by mike-sul — APPROVED** [review]

**Comment by doanac** [comment]
@mike-sul - this wasn't the branch i intended to push. i need to update and re-push

**Comment by doanac** [comment]
@mike-sul - this is good now that I've submitted the real diff, right?

**Comment by mike-sul** [comment]
@doanac LGTM

**Pull request merged**